lockbox-ios/Presenter/AccountSettingPresenter.swift

```diff
@@ -29,6 +29,21 @@ class AccountSettingPresenter {
         }.asObserver()
     }()
 
+    lazy private(set) var unLinkAccountTapped: AnyObserver<Void> = {
+        return Binder(self) { target, _ in
+            target.view?.displayAlertController(buttons: [
+                AlertActionButtonConfiguration(title: "Cancel",
+                                               tapObserver: nil,
+                                               style: .default),
+                AlertActionButtonConfiguration(title: "Unlink",
+                                               tapObserver: target.unlinkAccountObserver,
+                                               style: .destructive)],
+                                            title: "Are you sure you want to unlink your account?",
+                                            message: "This will sign you out and any settings you made will be reset for this device.",
```

Comment:
While this probably won't be a `line_length` violation when the string is added to `Constants.swift`, you should have no `swiftlint` warnings when compiling (or choose to manually ignore lines with `// swiftlint:disable:this [swiftlint_rule]`)